.github/patch-initial-checklist.md

```diff
@@ -10,33 +10,33 @@ The release pull request has been created! This checklist is a guide to follow f
-
-Additionally, make sure to differentiate between things in the testing notes that only apply to the feature plugin and things that apply when included in WooCommerce core as there may be variations there.
+## Create the Testing Notes
 
 * [ ] Run `npm ci`
 * [ ] Run `npm run package-plugin:deploy`. This will create a zip of the current branch build locally.
-* [ ] Create testing notes for the release. You can usually go through the pull requests linked in the changelog and grab testing notes from each pull.
+* [ ] Create the testing notes for the release. 
+  * [ ] For each pull request linked in the changelog grab the `User Facing Testing` notes from the PR's description. 
   * [ ] Add the notes to `docs/testing/releases`
   * [ ] Update the `docs/testing/releases/README.md` file index.
 * [ ] Copy a link to the release zip you created earlier into the testing notes. To generate the link you can upload the zip as an attachment in a GitHub comment and then just copy the path (without publishing the comment).
 * [ ] Commit and push the testing docs to the release branch.
 * [ ] Smoke test built release zip using the testing instructions you created:
-  * At least one other person should test the built zip - ping the current Rubik porter to be this person.
   * Test in a clean environment, e.g. Jurassic.Ninja site.
   * Test existing WooCommerce Blocks content works correctly after update (no block validation errors).
   * Test to confirm blocks are available and work correctly in oldest supported WordPress version (e.g. 5.3).
   * Confidence check - check blocks are available and function.
   * Test to confirm new features/fixes are working correctly.
   * Test any UI changes in mobile and desktop views.
   * Smoke test – test a cross section of core functionality.
+* [ ] Ask the porter of Rubik and Kirigami to review the changes as well and to approve the PR if everything looks good. Each porter is responsible for testing the PRs that fall under the focus of their own team. Shared functionality should be tested by both porters. This means that the Rubik porter will mostly test Cart & Checkout changes while Kirigami porter will test the other blocks.
+  * If all PRs are testing as expected, continue with the release.
+  * If one or more PRs are not testing as expected: ping the PR authors and the porter of the relevant team and ask them if the change is a release blocker or not (you can also ping the team lead if any of them is not available). In general, if it's not a regression or there is no product/marketing reason why that PR is a blocker, all other PRs should default to not being blockers.
+    * If there are blockers: stop the release and ask the PR author and team porter to fix them.
```

Comment:
I wonder in how far the team porter is a good fix for fixing a blocker. I was thinking that fixing a blocker would be the responsibility of the PR author only.

Reply:
Good point. I rewrote it in 329906d53cf925d91e6d8b9ba093e0d3c2b06635, let me know how it looks now! :slightly_smiling_face:

Reply:
Looks great now, @Aljullu!